parser/format/format.go

```diff
@@ -224,6 +224,7 @@ const (
 	RestoreStringWithoutDefaultCharset
 
 	RestoreTiDBSpecialComment
+	RestoreRemovePlacementRule
```

Comment:
The name of this flag `RestoreRemove...` is confusing.

IMHO a better name would be something like `SkipPlacementRule` / `SkipPlacementRuleForRestore`.

Reply:
Fix